packages/ui/src/components/input/date-time/index.tsx

```diff
@@ -37,16 +44,36 @@ export const DateTimeInput = forwardRef<HTMLInputElement, DateTimeInputProps>(
         const [anchorEl, setAnchorEl] = useState<HTMLInputElement | null>(null);
         const popoverRef = useRef<HTMLDivElement>(null);
         const [open, setOpen] = useState(false);
+        const [modal, setModal] = useState(false);
+
+        const resizeObserver = useRef(
```

Comment:
Not sure a ref is necessary here. If we declare the `ResizeObserver` in the `useEffect` hook below directly it should be the same (minus the ref we wouldn't use) as it's not used anywhere else.